moveit_core/collision_detection/src/collision_tools.cpp

```diff
@@ -273,7 +277,8 @@ void costSourceToMsg(const CostSource& cost_source, moveit_msgs::msg::CostSource
 void contactToMsg(const Contact& contact, moveit_msgs::msg::ContactInformation& msg)
 {
   msg.position = tf2::toMsg(contact.pos);
-  tf2::toMsg(contact.normal, msg.normal);
+  // tf2::toMsg(contact.normal, msg.normal);
+msg.normal =  tf2::toMsg(contact.normal);
```

Comment:
clang-format should catch this.

Reply:
https://github.com/ros-planning/moveit2/pull/40/commits/8152ab5f384fb97b227cca36d758d012a99d9a1e